src/test/ui/binop/binop-consume-args.stderr

```diff
@@ -35,10 +40,15 @@ error[E0382]: use of moved value: `lhs`
 LL | fn sub<A: Sub<B, Output=()>, B>(lhs: A, rhs: B) {
    |                                 --- move occurs because `lhs` has type `A`, which does not implement the `Copy` trait
 LL |     lhs - rhs;
-   |     --- value moved here
+   |     --------- value moved due to this method call
```

Comment:
this also could well be misleading -- maybe we should check the span to see if it's a desugared operator and right something about that?

Reply:
Checking the span seems kind of fragile to me - however, I don't think we keep track of the fact that the operator was desugared (as opposed to a literal `Shl::shl`) after typeck.

Reply:
Doesn't `DesugaredKind` hold that info? (It might not.)

I've encountered issues with operators before where I wanted to point at only the operator. It might make sense to add that span somewhere (but that could be done in a separate PR).